backend/main.py

```diff
@@ -375,9 +375,9 @@ def read_news(db=Depends(session_opener)):
 @app.get(
     "/api/v1/news/user_news"
 )
-def read_user_news(
+def get_user_upvoted_news(
```

Comment:
This is not the function to get the news that specific users have upvoted. The purpose of the function, according to the code we get, is that we get all news in the database "with" the value that if the user has upvoted the news